app/lib/matador/components/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from './dot';
+export * from './ui/Link';
```

Comment:
`/ui/Link` has been moved, so isn't this an error?

Reply:
`/ui/Link` has not been moved in my folder structure, so it's correct written like that